src/Pages/ResumeStorage/Components/FileButton.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { RiDeleteBin5Line } from 'react-icons/ri';
+import { FiDownload } from 'react-icons/fi';
+import styled from 'styled-components';
+import theme from '../../../Styles/theme';
+
+const FileButton = props => {
+  return (
+    <FileButtonBox type={props.type}>
```

Comment:
```suggestion
const FileButton = ({ type }) => {
  return (
    <FileButtonBox type={type}>
```
구조분해 할당 활용해줄 수 있습니다!

Reply:
넵 수정하였습니다!